sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -122,6 +122,44 @@ class SQLQuerySuite extends QueryTest with SharedSQLContext {
     checkKeywordsExist(sql("describe functioN abcadf"), "Function: abcadf not found.")
   }
 
+  test("unresolvable table-valued function") {
```

Comment:
Ah, I didn't find it since I hadn't rebased yet. Done, and added a test to plan parser suite too.